collectors/python.d.plugin/am2320/am2320.chart.py

```diff
@@ -0,0 +1,64 @@
+try:
```

Comment:
https://github.com/netdata/netdata/blob/380529c7cf217f62ffcc7347a93596c12d7fe90e/collectors/python.d.plugin/apache/apache.chart.py#L1-L5

please add the header

Reply:
move the header from the README.md

Reply:
done